base/usr/local/bin/configure-nonroot-gratia.py

```diff
@@ -30,6 +30,7 @@ def main():
     probe_config.attrib['Lockfile'] = '/var/lock/condor-ce/gratia.lock'
     probe_config.attrib['WorkingFolder'] = '/var/lib/condor-ce/'
     probe_config.attrib['DataFolder'] = '/var/lib/condor-ce/gratia/data/'
+    probe_config.attrib['CondorCEHistoryFolder'] = '/var/lib/condor-ce/gratia/data/'
```

Comment:
Is setting these two to the same dir safe?

Reply:
probably safe but also not really correct.  i've made some updates per our discussion